projects/kubernetes-sigs/image-builder/build/build_image_on_metal.sh

```diff
@@ -140,3 +142,4 @@ if [[ "$IMAGE_FORMAT" == "cloudstack" ]]; then
   OUTPUT_IMAGE_BLOB="*.qcow2"
 fi
 scp $SSH_OPTS $REMOTE_HOST:$REMOTE_HOME_DIR/$OUTPUT_IMAGE_BLOB $REPO_ROOT/$PROJECT_PATH/
+scp $SSH_OPTS $REMOTE_HOST:"$REMOTE_ARTIFACTS_PATH/$IMAGE_FORMAT/$IMAGE_OS/EKSD_MANIFEST_URL $REMOTE_ARTIFACTS_PATH/$IMAGE_FORMAT/$IMAGE_OS/KUBERNETES_VERSION" $REPO_ROOT/$ARTIFACTS_PATH/
```

Comment:
This should just be `ARTIFACTS_PATH`, it's an absolute path.
```suggestion
scp $SSH_OPTS $REMOTE_HOST:"$REMOTE_ARTIFACTS_PATH/$IMAGE_FORMAT/$IMAGE_OS/EKSD_MANIFEST_URL $REMOTE_ARTIFACTS_PATH/$IMAGE_FORMAT/$IMAGE_OS/KUBERNETES_VERSION" $ARTIFACTS_PATH/
```